uranium/packages/install_command.py

```diff
@@ -33,6 +43,12 @@ def populate_requirement_set(self, requirement_set, args, options,
                         options=options, session=session,
                         wheel_cache=wheel_cache, constraint=True
                 ):
+                    if packages_config and req.name in packages_config:
```

Comment:
is this check necessary, since getattr should always return some sort of dictionary?